libraries/session/src/main/java/androidx/media3/session/DefaultMediaNotificationProvider.java

```diff
@@ -374,6 +374,7 @@ private void ensureNotificationChannel() {
     NotificationChannel channel =
         new NotificationChannel(
             NOTIFICATION_CHANNEL_ID, NOTIFICATION_CHANNEL_NAME, NotificationManager.IMPORTANCE_LOW);
+    channel.setShowBadge(false);
```

Comment:
The linked issue (https://github.com/androidx/media/issues/131) and the title of this PR indicate you expect this to only affect API 26 and below, but this line is only executed on API 26 and above (since API < 26 will return early above on L372).

So is it more correct to describe this as a problem specific to API 26? If so, it might be clearer to gate this with `Util.SDK_INT == 26` with a comment.

Actually: I did some quick testing on a Pixel 2 emulator on different API levels, I see a badge on API 26 and 27, but not 28 - do you agree with that? In that case, let's gate it on `Util.SDK_INT == 26 || Util.SDK_INT == 27`

Reply:
Yes I share your observation.
I am not sure I understand why the code to setup the channel is not executed on ApI 26, because Channels were introduced with API 26...

Reply:
> I am not sure I understand why the code to setup the channel is not executed on ApI 26, because Channels were introduced with API 26...

Why do you believe that to be the case? This code is executed when `Util.SDK_INT < 26` is `false`, i.e. when `SDK_INT >= 26` is `true`, i.e. it **is** executed on API 26.